cava.tera

```diff
@@ -2,13 +2,15 @@
 whiskers:
   version: 2.0.0
   matrix:
+    - variant: ["", "-transparent"]
     - flavor
-  filename: "themes/{{ flavor.identifier }}.cava"
+  filename: "themes/{{ flavor.identifier }}{{ variant }}.cava"
```

Comment:
```suggestion
  filename: "themes/{{ flavor.identifier }}{% if variant == "transparent" %}-{{ variant }}{% endif %}.cava"
```

Reply:
I did try that before opening the PR, but got an error from Whiskers. It seems like using conditional blocks inside the `filename` field is not supported?

```sh
$ whiskers cava.tera
Error: Frontmatter is invalid

Caused by:
    Invalid YAML frontmatter (L6:62) : did not find expected key at line 6 column 62, while parsing a block mapping at line 2 column 3
```

```sh
$ whiskers --version
catppuccin-whiskers 2.1.1
```

Reply:
Ah, remove the quotes.

Reply:
It does, thanks! I've pushed the suggested changes

Reply:
Oh btw, do you think this quotes thing can be considered an bug in Whiskers, and should I report it ? Or is it supposed to error out like it does?